package.json

```diff
@@ -74,6 +77,7 @@
     "debounceify": "^1.1.0",
     "drive-bundler": "^1.0.0",
     "framed-stream": "^1.0.1",
+    "fs": "npm:bare-node-fs",
```

Comment:
don't do this on platform - use the right thing in the right place and isBare from which-runtime module to determine in cases where it's both

Reply:
(also, generally,  the preferred way to do this is import maps)

Reply:
@davidmarkclements Ah, right now this is a workaround for the `which` dependency within bare-dev repo which is yet to be replaced.

https://holepunchdev.slack.com/archives/C054FE134P6/p1710861131438619?thread_ts=1710850156.778759&cid=C054FE134P6

Reply:
Tried using add the import map as well but it didn't work for bare-dev.

Reply:
This should now be removed.